magicbot/magicrobot.py

```diff
@@ -60,18 +62,23 @@ class MagicRobot(wpilib.RobotBase):
     #: If True, teleopPeriodic will be called in autonomous mode
     use_teleop_in_autonomous = False
 
-    def robotInit(self):
-        """
-            .. warning:: Internal API, don't override; use :meth:`createObjects` instead
-        """
+    def __init__(self) -> None:
```

Comment:
Why move this to __init__?

Reply:
So that we can avoid calling robotInit in tests.